src/client/views/cube-view/cube-view.tsx

```diff
@@ -567,113 +561,114 @@ export class CubeView extends React.Component<CubeViewProps, CubeViewState> {
       openUrlShortenerModal={this.openUrlShortenerModal}
       openDruidQueryModal={this.openDruidQueryModal}
       customization={customization}
-      getDownloadableDataset={() => this.downloadableDataset}
       changeTimezone={this.changeTimezone}
       updatingMaxTime={updatingMaxTime}
     />;
 
     return <CubeContext.Provider value={this.getCubeContext()}>
-      <div className="cube-view">
-        <GlobalEventListener resize={this.globalResizeListener}/>
-        {headerBar}
-        <div className="container" ref={this.container}>
-          <PartialTilesProvider>{({ series, filter, addFilter, addSeries, removeTile }) => <React.Fragment>
-            {!layout.factPanel.hidden && <DimensionMeasurePanel
-              style={styles.dimensionMeasurePanel}
-              clicker={clicker}
-              essence={essence}
-              menuStage={menuStage}
-              addPartialFilter={dimension =>
-                addFilter(dimension, DragPosition.insertAt(essence.filter.length()))}
-              addPartialSeries={series =>
-                addSeries(series, DragPosition.insertAt(essence.series.count()))}
-            />}
-            {!this.isSmallDevice() && !layout.factPanel.hidden && <ResizeHandle
-              direction={Direction.LEFT}
-              value={layout.factPanel.width}
-              onResize={this.onFactPanelResize}
-              onResizeEnd={this.onPanelResizeEnd}
-              min={MIN_PANEL_WIDTH}
-              max={MAX_PANEL_WIDTH}
-            >
-              <DragHandle/>
-            </ResizeHandle>}
-
-            <div className="center-panel" style={styles.centerPanel}>
-              <div className="center-top-bar">
-                <div className="dimension-panel-toggle"
-                     onClick={this.toggleFactPanel}>
-                  <ToggleArrow right={layout.factPanel.hidden}/>
-                </div>
-                <div className="filter-split-section">
-                  <FilterTilesRow
-                    locale={customization.locale}
-                    timekeeper={timekeeper}
-                    menuStage={visualizationStage}
-                    partialFilter={filter}
-                    removePartialFilter={removeTile}
-                    addPartialFilter={addFilter}
-                  />
-                  <SplitTilesRow
-                    clicker={clicker}
-                    essence={essence}
-                    menuStage={visualizationStage}
-                  />
-                  <SeriesTilesRow
-                    removePartialSeries={removeTile}
-                    partialSeries={series}
-                    menuStage={visualizationStage}
-                    addPartialSeries={addSeries}/>
+      <DownloadableDatasetProvider>
```

Comment:
What has changed in this block?

Reply:
`DownloadableDatasetProvider` wraps `<div.cube-view>`, so indentation changed in result.